meson.build

```diff
@@ -1,3 +1,3 @@
 project('appstream-data-pantheon')
 
-run_script = run_command('bash', meson.project_name() + '.sh')
+run_script = run_command('bash', meson.project_name() + '.sh', check: false)
```

Comment:
This script doesn't even exist, so it always fails. What's it supposed to do? What's the result of running Meson?

The script was removed in commit b616c8ca253f7b1a6a528350350f0d092353bea6 for the record. Setting this to `check: true` would reveal that failure. ;)

Reply:
Haha, that's what I get by somewhat blindly scanning the repos for the run_command issue without fully understanding the repo first. 😅️
Looking at the commit you mentioned, it seems to me, as if the entire meson part is not needed anymore. Do you think it would be good to just remove the meson.build file?

Reply:
Probably?

Either that or reimplement it from scratch to install the relevant contents of pantheon-data/main/